File: Javascript/6KYU/FizzBuzz.js
function solution(number){
  a=0;b=0;c=0
  for(x=1;x<number;x++){
    if(x%15==0){c++}
    else if(x%5==0){b++}
    else if(x%3==0){a++}
  }
  return [a,b,c]
}

File: Python/6KYU/FizzBuzz.py
def solution(number):
    a=0;b=0;c=0
    for x in range(1,number):
        if x%15==0:
            c += 1
        elif x%5==0:
            b += 1
        elif x%3==0: 
            a += 1
    return [a,b,c]